bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/preferences/GlobalMcuPage.java

```diff
@@ -99,5 +107,19 @@ protected void createFieldEditors() {
 		addField(folder);
 	}
 
+	@Override
+	public boolean performOk()
+	{
+		boolean result = super.performOk();
+		String openOcdPath = fPersistentPreferences.getInstallFolder(StringUtil.EMPTY);
+		Path path = Paths.get(openOcdPath);
+		if (!openOcdPath.isBlank())
+		{
+			path = path.getParent().resolve("share").resolve("openocd").resolve("scripts"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
Seems good just one small thing
I think we can take care of this warning which is very valid for the given line.
<img width="530" alt="image" src="https://user-images.githubusercontent.com/85216275/226910431-7e6f9f94-d740-4319-a6a7-b8d7207565e6.png">

Reply:
fixed